docs/installation.md

```diff
@@ -30,10 +30,20 @@ you can install alphaDIA via `pip`.
 Please make sure you have a valid installation of conda or miniconda.
 We recommend setting up miniconda as described on their [website](https://docs.conda.io/projects/miniconda/en/latest/).
 
-If you want to use `.raw` files on Thermo instruments alphaRaw is required, which depends on Mono. A detailed guide to installing alphaRaw with mono can be found [here](https://github.com/MannLabs/alpharaw#installation).
+If you want to use `.raw` files on Thermo instruments alphaRaw is required, which depends on Mono.
+A detailed guide to installing alphaRaw with mono can be found [here](https://github.com/MannLabs/alpharaw#installation), a short version is given
+[below](#2-installing-mono).
 
 ### 2. Setting up the environment
 
+It is highly recommended to use a conda environment for the installation of alphaDIA.
+This ensures that all dependencies are installed correctly and do not interfere with other packages.
+```bash
+conda create -n alphadia python=3.11
```

Comment:
Let's change the default to `alpha`. This is the default environment name expected by the dev install of the gui.

Reply:
could we set "envName": "alphadia" instead in the gui install?

Reply:
Absolutely 👍